lib/component.js

```diff
@@ -255,6 +256,29 @@ class Component extends WebComponent {
     } else {
       this.el = this;
     }
+
+    const defaultContexts = this.getConfig(`defaultContexts`);
+    if (!(defaultContexts instanceof Map)) {
```

Comment:
I'm not sure what the perf overhead for context related checking is. Since it happens on constructor for every panel component, even though the panel component may have nothing to do with contexts.

Reply:
I can probably replace this particular `instanceof` with `defaultContexts.constructor.name === "Map"`. The other `instanceof` checks only run if the component has contexts. That being said, `instanceof` is a relatively slow operation as I understand it, although I'm not sure how noticeable it is.